framgia/ph_sels_rails_johnedel
# Creation of Admin Categories

PR description:  Creation of Admin Categories
Content: CRUD for categories
[Commands]
rails db:migrate:reset db:seed
rails db:server

[Pre-conditions]
Admin User
email: example@railstutorial.org
password: password

Normal User
email: example@railstutorial.org
password: password

To test the module:
Login as admin
1. Go to account there should be categories if you logged in as admin and if not there should not be categories
2. Now explore the crud functionalities for the Category

[Expected Output]
Admin users can CRUD the categories
			
[Notes]
Words is still not working
